poor/i18n.py

```diff
@@ -30,3 +30,19 @@
 def _(message):
     """Return the localized translation of `message`."""
     return _translation.gettext(message)
+
+def __(message, language):
+    """Return the translation of `message` to `language`. Note that the same
+    notation is used for languages, as in the voicecommand module"""
+    # exceptions
+    l = {
+        "de": "de_DE",
+        "en": "en_US",
+        "en-US": "en_US",
+        "en-US-x-pirate": "en_US",
+        "es": "es_ES",
+        }
+    return gettext.translation( "poor-maps",
+                                localedir=poor.LOCALE_DIR,
+                                languages=l.get(language, language),
+                                fallback=True ).gettext(message)
```

Comment:
I must say I don't understand this suggestion

Reply:
`gettext.translation` creates an instance of the `Translations` class. You're recreating the effectively same instance on every call. While I don't know how slow it is, it's likely scanning the locale directories on every instantiation, it'd nicer to avoid it.

Reply:
That's bad - didn't think about it. I made the change